examples/kotlin/build.gradle

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
+plugins { id 'org.apache.beam.module'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.21'
+}
+
+applyJavaNature(exportJavadoc: false)
```

Comment:
We should separate out a KotlinNature and set up e.g. Spotless to use ktlint. (separately, instead of all these "natures" in one plugin, they should be separate BeamKotlinPlugin etc but that is another big step)

Reply:
Yeah, +1 for ktlint.
I'll try to set it up for the project once this is merged but as you mentioned, it could be a significant refactor